### PR TITLE
[Merged by Bors] - chore: better .gitignore

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
-/build
-/lake-packages
-/.cache
```
Comment (eric-wieser) [inline .gitignore:3]: When did we stop using `.cache`?

**Review by eric-wieser — APPROVED** [review]
bors d+

File: .gitignore (hunk 1 above)
Comment (digama0) [reply .gitignore:3]: `.cache` is still used, when the user does not have a home directory as detected via env vars.

Comment (eric-wieser) [reply .gitignore:3]: I added it back along with a speculative comment

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+# macOS leaves these files everywhere:
 .DS_Store
+# Created by `lake exe cache get` in unusual configurations
```
Comment (semorrison) [inline .gitignore:3]:
```suggestion
# Created by `lake exe cache get` if no home directory is available
```

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**